diff --git a/node_modules/react-native-amap3d/lib/ios/maps/AMapPolyline.m b/node_modules/react-native-amap3d/lib/ios/maps/AMapPolyline.m
index de2d8bc..8c62a3c 100644
--- a/node_modules/react-native-amap3d/lib/ios/maps/AMapPolyline.m
+++ b/node_modules/react-native-amap3d/lib/ios/maps/AMapPolyline.m
@@ -72,11 +72,6 @@ - (void)setColors:(NSArray *)colors {
     [_polyline setDrawStyleIndexes:@[@(1), @(2)]];
 }
 
-- (void)setDashed:(BOOL)dashed {
-    _dashed = dashed;
-    _renderer.lineDash = dashed;
-}
-
 - (void)setGradient:(BOOL)gradient {
     _gradient = gradient;
     _renderer.gradient = gradient;
@@ -99,7 +94,6 @@ - (MAOverlayRenderer *)renderer {
         _renderer.lineWidth = _width;
         _renderer.strokeColor = _color;
         _renderer.strokeColors = _colors;
-        _renderer.lineDash = _dashed;
         _renderer.gradient = _gradient;
     }
     return _renderer;
diff --git a/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m b/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m
index 66e4108..586d19c 100644
--- a/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m
+++ b/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m
@@ -66,11 +66,6 @@ - (void)setLocationStyle:(LocationStyle *)locationStyle {
     _locationStyle.strokeColor = locationStyle.strokeColor;
     _locationStyle.lineWidth = locationStyle.strokeWidth;
     _locationStyle.image = locationStyle.image;
-    _locationStyle.locationDotBgColor = locationStyle.locationDotBgColor;
-    _locationStyle.locationDotFillColor = locationStyle.locationDotFillColor;
-    _locationStyle.enablePulseAnnimation = locationStyle.enablePulseAnnimation;
-    _locationStyle.showsHeadingIndicator = locationStyle.showsHeadingIndicator;
-    _locationStyle.showsAccuracyRing = locationStyle.showsAccuracyRing;
 
     [self updateUserLocationRepresentation:_locationStyle];
 }
