diff --git a/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m b/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m
index 66e4108..586d19c 100644
--- a/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m
+++ b/node_modules/react-native-amap3d/lib/ios/maps/AMapView.m
@@ -66,11 +66,6 @@ - (void)setLocationStyle:(LocationStyle *)locationStyle {
     _locationStyle.strokeColor = locationStyle.strokeColor;
     _locationStyle.lineWidth = locationStyle.strokeWidth;
     _locationStyle.image = locationStyle.image;
-    _locationStyle.locationDotBgColor = locationStyle.locationDotBgColor;
-    _locationStyle.locationDotFillColor = locationStyle.locationDotFillColor;
-    _locationStyle.enablePulseAnnimation = locationStyle.enablePulseAnnimation;
-    _locationStyle.showsHeadingIndicator = locationStyle.showsHeadingIndicator;
-    _locationStyle.showsAccuracyRing = locationStyle.showsAccuracyRing;
 
     [self updateUserLocationRepresentation:_locationStyle];
 }
